eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -219,15 +160,44 @@ jobs:
         - pwsh: npm install -g moxygen
           displayName: Install Moxygen to generate markdown for docs.microsoft.com
 
+        - pwsh: |
+            $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
```

Comment:
We should not be pulling the lastest version automatically. The version we depend on should be hard coded and updated when we update the parser. We need to keep the version pinned to what is currently working in the repo.